internal/config/application.go

```diff
@@ -291,3 +299,18 @@ func loadConfig(v *viper.Viper, configPath string) error {
 	}
 	return nil
 }
+
+func checkDefaultSourceValues(source string) error {
+	valid := map[string]struct{}{
+		"registry": {},
+		"docker":   {},
+		"podman":   {},
+		"":         {},
+	}
+
+	if _, ok := valid[source]; !ok {
+		return fmt.Errorf("%s is not a valid default source; please use one of ['registry', 'docker', podman', '']", source)
```

Comment:
in case the list of valid values changes, the string here should be built from the same variable/source of truth to stay consistent.